File: Codes in Python3/Graph/BFS.py
from typing import List
from queue import Queue
class Solution:
    #Function to return Breadth First Traversal of given graph.
    def bfsOfGraph(self, V: int, adj: List[List[int]]) -> List[int]:
        bfs=[]
        vis=[False for i in range(V)]
        q=[0]
        vis[0]=True
        while q:
            x=q.pop(0)
            bfs.append(x)
            for i in adj[x]:
                if vis[i]==False:
                    vis[i]=True
                    q.append(i)
        return bfs

File: Codes in C++/Graph/BFS.cpp
class Solution {
  public:
    // Function to return Breadth First Traversal of given graph.
    vector<int> bfsOfGraph(int V, vector<int> adj[]) {
       vector<int> vis(V,0);
       vector<int> bfs;
       queue<int> q;
       vis[0]=1;
       q.push(0);
       while(!q.empty()){
           int x=q.front();
           q.pop();
           bfs.push_back(x); 
           for(auto it:adj[x]){
           if(vis[it]==0)
           {
            vis[it]=1;
            q.push(it);
           }
           }
       }
       return bfs;
    }
};